src/Behat/Mink/Driver/CoreDriver.php

```diff
@@ -90,6 +96,28 @@ public function getContent()
      */
     public function find($xpath)
     {
+        $elements = array();
+
+        foreach ($this->findElements($xpath) as $xpath) {
+            $elements[] = new NodeElement($xpath, $this->session);
+        }
+
+        return $elements;
+    }
+
+    /**
+     * Finds elements with specified XPath query.
+     *
+     * @see find()
+     *
+     * @param string $xpath
+     *
+     * @return string[] The XPath of the matched elements
+     *
+     * @throws UnsupportedDriverActionException When operation not supported by the driver
+     */
+    protected function findElements($xpath)
```

Comment:
So it finds xpaths of elements within element with given xpath. We don't have much words to operate with here. Only `find`, `element`, `xpath`, but since `xpath` is implementation detail we can't use it method name.

Maybe we can call it `findElementSelectors`.